Mathlib/CategoryTheory/IsConnected.lean

```diff
@@ -259,6 +259,10 @@ theorem isPreconnected_of_isPreconnected_op [IsPreconnected Jᵒᵖ] : IsPreconn
 theorem isConnected_of_isConnected_op [IsConnected Jᵒᵖ] : IsConnected J :=
   isConnected_of_equivalent (opOpEquivalence J)
 
+variable (J) in
+theorem isConnected_iff_isConnected_op : IsConnected J ↔ IsConnected Jᵒᵖ :=
```

Comment:
I would prefer `IsConnected Jᵒᵖ ↔ IsConnected J`, and we may consider making it a simp lemma.